api/tests.py

```diff
@@ -51,3 +51,17 @@ def testReminderWithArraignmentToday(self):
         resp_json = json.loads(response.content)
         self.assertEqual(Alert.objects.all().count(), 0)
         self.assertEqual(resp_json['status'], '410 Gone')
+   
+
+    def testPreventDuplicateReminder(self):
+        arraignment_datetime = (datetime.today() + timedelta(days=8)).strftime('%Y-%m-%dT%H:%M:%S')
+        request = self._post('/api/reminders', {
+            "arraignment_datetime": arraignment_datetime,
+            "case_num": 1,
+            "phone_num": "+1-918-555-5555",
+        })       
+        response = reminders(request)
+        reminders(request) # Submitting a duplicate reminder
```

Comment:
Move this below the assertions, and then copy the assertions after this 2nd call.